doc/about/support.md

```diff
@@ -54,23 +69,21 @@ In addition to developing and maintaining the many open source tools built by th
 - **Personnel to operate mybinder.org**: Running a heavily-used and complex cloud service requires expertise in cloud infrastructure development and operation.
   We receive in-kind contributions from several individuals and organizations to help us sustain these operations.
 
-## Help with our open source efforts
+## Join our team
+
+There are many ways that you can contribute to the Binder project, and the Binder community welcomes any and all contributions that others are willing to give.
+
+To get started contributing, check out [the JupyterHub Team Compass guide to contributing](https://jupyterhub-team-compass.readthedocs.io/en/latest/team/skills.html).
 
-Finally, all of the techology behind mybinder.org is built by an open community.
-We welcome any kind of contributions and support.
-You can find many ways to contribute to the Binder community, codebase, and services at [](./contribute.md).
+Here are a few ways that you can begin contributing to the Binder project as an individual.
 
-## Attribution for team members
+### Contribute to the Binder documentation
 
-We wish to recognize the contributions that individuals and organizations make on behalf of Binder.
-For this reason, we define a few key contribution areas below, and explicitly recognize these contributions in our mybinder.org pages.
+This repository is a "meta-repository" for the Binder user documentation. If you notice anything missing, or feel that it could be improved, please [open an issue to discuss](https://github.com/jupyterhub/binder/issues/new/choose).
 
-- *mybinder.org federation members* - Institutions that deploy, fund, and staff their own BinderHub in the federation.
-- *mybinder.org partner institutions* - Institutions that employ individuals that operate any hubs in the mybinder.org federation (if not full federation members).
-- *mybinder.org major supporters* - Institutions that provide significant financial resources or credits for any hubs in the mybinder.org federation ("significant" is defined as anything greater than $10,000 annually).
+Check out [the documentation contribution guide](https://github.com/jupyterhub/binder/blob/master/CONTRIBUTING.rst) for more
+information.
 
-## Previous grants and major support
+### Contribute to the codebase of the Binder Project
 
-- From 2019 through 2021, `gke.mybinder.org` was supported by cloud credits from [Google Cloud](https://cloud.google.com/).
-- In 2017 the Binder Project received a 1-year grant from the
-`Moore Foundation <https://figshare.com/s/e9d0ad7bdc4e405cccfa>`_.
+Most of the code used by the Binder Project is not hosted in this repository. If you'd like to contribute to the code, check out [the JupyterHub Team Compass guide to contributing](https://jupyterhub-team-compass.readthedocs.io/en/latest/team/skills.html).
```

Comment:
The skills link is included in the above section, so might be good to link to relevant repositories, or perhaps indicate that https://github.com/jupyterhub/binderhub is the starting point?